src/library.jsx

```diff
@@ -0,0 +1,64 @@
+
+const Box = (props) => {
```

Comment:
This would be easier to manage if you used a prop called `size` instead of passing a height and width. It's also what was asked in the assignment description.